### ISSUE TITLE
Basic exact match not working for benzene C1=CC=CC=C1

### ISSUE THREAD

**Issue by ThomasLabstep** [epam/Indigo#546]
```
class TestClient:
    def setup_method(self):
        loadFixtures()

        data = 'C'
        push(data)

        data = 'C1CCCCC1'
        push(data)

        data = 'C1=CC=CC=C1'
        push(data)

        data = 'N'
        push(data)

        time.sleep(2)

    def test_search_exact(self):
        result = search('exact', 'C1=CC=CC=C1')
        assert len(list(result)) == 1

        expectedResult = {
            '3': 'C1=CC=CC=C1'
        }
        assert result == expectedResult
```

When trying to debug this exact match we notice that the following code is discarding the results:
```
        for el_response in res.get("hits", {}).get("hits", []):
            record = get_record_by_index(el_response, self.index_name)
            for action_fn in postprocess_actions:
                record = action_fn(record, indigo_session)
                if not record:
                    continue
            yield record
```
I could confirm it's going into 'if not record'.

Any idea why?
It's working fine with other molecules like 'N', 'C', 'C1CCCCC1'

Thank you for your help :)

**Comment by MysterionRise** [comment]
Let me take a look at this @ThomasLabstep

**Comment by ThomasLabstep** [comment]
@MysterionRise Any update on this issue?

**Comment by MysterionRise** [comment]
@ThomasLabstep sorry about that delay, will try to come back to this one soon

**Comment by MysterionRise** [comment]
@ThomasLabstep  it looks like we got a bug in exact match, would be fixed most likely with #644 stay tuned!

**Comment by ThomasLabstep** [comment]
All good now!
